frontend/lang/messages/he-IL.json

```
@@ -79,7 +79,8 @@
     "tag-events": "אירועי תגיות",
     "category-events": "אירועי קטגוריות",
     "when-a-new-user-joins-your-group": "כאשר משתמש חדש מצטרף לקבוצה",
-    "recipe-events": "אירועי מתכון"
+    "recipe-events": "אירועי מתכון",
+    "label-events": "אירועי תווית"
```

```suggestion
    "recipe-events": "אירועי מתכונים",
    "label-events": "אירועי תוויות"
```

Small fixes for consistency. I noticed that most other "x-events" used the plural form of the noun.